src/app/app.component.css

```diff
@@ -0,0 +1,19 @@
+app-advertisments-list {
```

Comment:
Remove this
home page story covers this.

Reply:
Done